src/_plugins/code_tabs.rb

```diff
@@ -29,20 +36,17 @@
 #   ```objectivec
 #   // Implementation file code
 #   ```
-#   {% sample Swift %}
-#   <?code-excerpt "MyVC.swift" title?>
-#   ```swift
-#   // Swift code here
-#   ```
 #   {% endsamplecode %}
 #
-# The "samplecode" tag accepts a single parameter that represents the title of this sample. The
```

Comment:
Should we really be changing the line length of comments for this .rb file?

Reply:
Yeah, I get your point. At first I thought I had to fix the problem in this file and I have a hard time seeing when all the lines wrap.